holidays/countries/hongkong.py

```diff
@@ -198,8 +198,16 @@ def _populate(self, year):
         dt = self.cnls.lunar_to_gre(year, 8, 15)
         mid_autumn_date = date(dt.year, dt.month, dt.day)
         if self.observed:
-            if mid_autumn_date.weekday() == SAT:
+            # if Chinese Mid-Autumn Festival lies on Saturday
+            # from 1983 to 2010 public holiday lies on same day
+            # since 2011 public holiday lies on Monday
+            if mid_autumn_date.weekday() == SAT and year < 2011:
```

Comment:
What was the rule for Mid-Autumn Festival on Saturday **before** 1983? Same as after 2010?

Reply:
Updated please check